recipes/linearmodels/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "linearmodels" %}
+{% set version = "4.17" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 29f07c73be57b5356da41cad83ae53604b215b00cbf3e61347fe5d434cdcfae2
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```

Comment:
You probably need a `build` section

Reply:
Ah. I have now copied the one from the pandas recipe. Hopefully that is sufficient?